conf/solr/8.8.1/schema_dv_mdb_fields.xml

```diff
@@ -39,6 +39,7 @@
     <field name="coverage.Temporal" type="text_en" multiValued="true" stored="true" indexed="true"/>
     <field name="coverage.Temporal.StartTime" type="text_en" multiValued="true" stored="true" indexed="true"/>
     <field name="coverage.Temporal.StopTime" type="text_en" multiValued="true" stored="true" indexed="true"/>
+    <field name="creator" type="text_ws" multiValued="true" stored="true" indexed="true"/>
```

Comment:
Let's revert this Solr schema change. The Solr schema shouldn't change for this pull request because anyone wanting to play with the demo tsv can update their schema like normal (manually or using the script provided in the customizing metadata section of the admin guide).